Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+django = "*"
+python-decouple = "*"
+
+[dev-packages]
+
+[requires]
+python_version = "3.6"
```

Comment:
It is a good practice to have python_version = "3", so that it will work with all the 3. python version, instead of hard coding it to use only 3.6

Reply:
@noshaikh you can respond here.